### PR TITLE
App Branding

### PR DESCRIPTION
**Pull request by erdemyerebasmaz**
[Run CI](https://github.com/breez/l-breez/actions/workflows/CI.yml)

This PR addresses
- #13 

Various assets are updated to give Liquid app it's own identity.

### Changelist:
- Updated App logo
  - and it's color on App Bar(Light Theme only)
- Update Drawer footer image
- Applied theme changes to splash screen on Android
- Apply Breez theme on system navigation bar

### CODE REVIEW

File: android/app/src/main/res/mipmap-mdpi/ic_launcher.png
Comment (dangeross) [inline android/app/src/main/res/mipmap-mdpi/ic_launcher.png:1]: I would argue this is not going to be clearly visible on a device. Maybe just the **B**💧 is enough

**Review by dangeross — COMMENTED** [review]
There seems to be an inconsistency between the base and round/foreground images.

**Comment by erdemyerebasmaz** [comment]
> There seems to be an inconsistency between the base and round/foreground images.

Round icon is used for app icon on launcher, foreground image is used on splash screen whenever possible.

**Comment by erdemyerebasmaz** [comment]
@dangeross I've simplified resources, splash & launcher values are separated properly with no overlaps.

**Review by dangeross — APPROVED** [review]

**Pull request merged**